### ISSUE TITLE
A770 crashing when opening a Microsoft Store app (UWP)

### ISSUE THREAD

**Issue by Denizeri24** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#876]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Application is not cracked, modded and use the latest patch

### Application [Required]

Disney+, Apple Music

### Processor / Processor Number [Required]

8700K

### Graphic Card [Required]

A770 16 GB

### GPU Driver Version [Required]

32.0.101.6314

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [ ] DirectX12
- [ ] DirectX11
- [ ] DirectX10
- [ ] DirectX9
- [X] Not applicable

### Windows Build [Required]

Windows 11 24H2

### Intel System Support Utility report

[result.txt](https://github.com/user-attachments/files/18042205/result.txt)

### Description and steps to reproduce [Required]

Just open a movie from Disney+ or open Apple Music, monitor will going blacks for ~3 seconds, 2 times and windows's monitor settings will be reset like limiting 120 Hz (normally 240 Hz) and color bit to 6 (normally 12) also colors going darker. Only power on + off monitor for resetting back to normal screen settings. (tried HDMI too, two different 2.1 cable and result same.)

321URX via DP 1.4 (Tested with two DP 2.1 cable, same result)

### Device / Platform

Latest Windows 11 (not preview)

**Comment by Denizeri24** [comment]
@Zack-Intel 

idk earlier drivers but at least last 6 month and yes, this happened with earlier drivers.

no, not only these apps. I asked to intel arturo and he get same crash with opening Spotify.

msi mpg 321urx and asus pg258q

**Comment by Arturo-Intel** [comment]
@Denizeri24 Do you have Adaptive Sync turned on in your monitor settings?

**Comment by Denizeri24** [comment]
@Arturo-Intel no, my monitor have amd freesync but i cant turn on because i using via displayport. only i can turn on via hdmi but arc doesnt support 4k 240hz via hdmi

**Comment by Arturo-Intel** [comment]
I will close this for now.
